sdk/core/core-http/review/core-http.api.md

```diff
@@ -525,6 +531,7 @@ export function parseXML(str: string, opts?: SerializerOptions): Promise<any>;
 
 // @public
 export interface PipelineOptions {
+    clientRequestIdOptions?: ClientRequestIdOptions;
```

Comment:
is this a common consumer scenario? Should we make it an `InternalPipelineOptions` first?

Reply:
This is my main question too - when would a consumer use this? I want to be very sure we want this to be public surface area.

Reply:
Yeah I think this would be better off as an internal property, From the generator standpoint we don't need to have it publicly exposed.